google/cloud/storage/bucket_metadata.h

```diff
@@ -136,50 +136,51 @@ std::ostream& operator<<(std::ostream& os, CorsEntry const& rhs);
  * @warning this is a Beta feature of Google Cloud Storage, it is not subject
  *     to the deprecation policy and subject to change without notice.
  *
- * @see Before enabling Bucket Policy Only please review the
- *     [feature documentation][bpo-link], as well as
+ * @see Before enabling Uniform Bucket Level Access please
```

Comment:
Please update these to link to the UBLA documentation instead of the BPO documentation

Reply:
Can you give us a link?  Usable outside google?

Reply:
And for what is worth, there is already a bug (#3193) to fix that once the links are available.